resources/sass/app.scss

```
@@ -230,7 +228,7 @@ dl.import-meta dd:not(:last-child) {
 
 dl.import-meta .download-csv {
     float: right;
-    @media(max-width: $width-breakpoint-tablet){
+    @media(max-width: $max-width-breakpoint-tablet){
```

$max-width-breakpoint-mobile
```suggestion
    @media(max-width: $max-width-breakpoint-mobile){
```